about.html

```diff
@@ -0,0 +1,136 @@
+ <title> ABOUT</title>
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="styles.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+ <!-- ABOUT -->
+<!-- HAMBURGER-WINDOW start-->
+
+<div  class="hambu-wind not-active"> 
+ <div class="x-btn"> 
+   <button type="btn-close" id="close">
+     <img src="/bg-img/blackclose.png" alt="x">
+   </button>
+ </div>
+ <a class="pop-window" href="/index.html">Home</a>
+ <a class="pop-window" href="#about">Program</a>
+ <a class="pop-window" href="#footer"> Join</a>   
+ <a class="pop-window" href="#footer"> Sponsors</a>  
+ <a class="pop-window" href="#footer"> News</a>  
+</div>
+
+<!-- HAMBURGER-WINDOW end-->
+  <!-- SECTION 1 -->
+ <section id="section-about">
+  <div id="about-div">
+    <div class="social-media">
+      <i class="fa-brands fa-square-facebook fa-2x"></i>
+      <i class="fa-brands fa-square-twitter fa-2x"></i>
+    </div>
+    <nav id="navbar">
+      <div id="nav-content"> 
+        <a id="ham-link" >
+        <img id="ham-btn" src="/bg-img/Hamburger_icon.svg.png" alt="hamburger-icon">
+        </a>
+        <a id="logo-mobile"> <img id="logo-btn-show" src="/bg-img/logo.png" alt="logo-icon"> </a>
+        <a id="logo-link" href="/index.html">
+          <img id="logo-btn" src="/bg-img/logo.png" alt="logo-icon">
+          </a>
+        <div id="nav-pages">
+          <a class="pop-window" href="/about.html">About</a>
+          <a class="pop-window" href="#about">Program</a>
+          <a class="pop-window" href="#footer"> Join</a>   
+          <a class="pop-window" href="#footer"> Sponsors</a>  
+          <a class="pop-window" href="#footer"> News</a>  
+        </div>
+        <div id="contact"> 
+          <button id="contact-btn" > Contact </button>
+        </div>
+      </div>
+    </nav>
+   <div id="about-bg"> 
+    <div class="title-about">
+     <h1> GET READY FOR THE ULTIMATE FESTIVAL WINTER 2023 </h1>
+    </div>
+    <div class="paragraph" id="about-p"> Lolla takes over historic Grant Park in Downtown Chicago, alongside the beautiful Chicago skyline and the shores of Lake Michigan. Take advantage of interactive areas 
+     with giveaways, merch and more!</div>
+    <div class="date" >SAVE THE DATE! MARCH 2-5-2023
+     <p>@ Chicago, Illinois, USA.</p>
+    </div>
+   </div>
+  </div>
+ </section>
+
+ <!-- SECTION 2 -->
+
+ <section id="section-2">
+  <div id="about-bg-2"> 
+   <div class="title-about-2">
+    <h1> ULTRA-PALLOZA LOGO <div id="program-separator-line" class="guide-bar"></div></h1>
+   </div>
```

Comment:
- [ ] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).